compiler/src/iree/compiler/Preprocessing/Common/ApplyPDLPatterns.cpp

```diff
@@ -389,6 +446,11 @@ static LogicalResult rewriteAsFlowDispatch(
                                        "unhandled operand/result types");
   }
   StringAttr externalFnNameAttr = dyn_cast<StringAttr>(externalFnName);
+
+  std::string uniqueExternalFnName =
+      makeUniqueFuncName(externalFnNameAttr.getValue().str(), inputTypes,
+                         resultTypes, otherOperandTypes);
+  externalFnNameAttr = rewriter.getStringAttr(uniqueExternalFnName);
```

Comment:
echoing earlier comment. We just want the executable name to account for operand types... we dont need to change the entry point name....